consensus/src/marshal/actor.rs

```diff
@@ -549,6 +492,26 @@ impl<
                             // an in-process block from being processed that is below the new sync floor
                             // updating `last_processed_height`.
                             self.pending_ack = None.into();
+
+                            // Prune the finalized block and finalization certificate archives in parallel.
+                            if let Err(err) = try_join!(
+                                // Prune the finalized blocks archive
+                                async {
+                                    self.finalized_blocks.prune(height).await.map_err(Box::new)?;
+                                    Ok::<_, Box<dyn Error + Send + Sync>>(())
+                                },
+                                // Prune the finalization certificate archive
+                                async {
+                                    self.finalizations_by_height
+                                        .prune(height)
+                                        .await
+                                        .map_err(Box::new)?;
+                                    Ok::<_, Box<dyn Error + Send + Sync>>(())
```

Comment:
Can we create an alias for `Box<dyn Error + Send + Sync>` in `utils` and use it to replace `anyhow`?